VMBackup/main/Utils/HandlerUtil.py

```diff
@@ -780,6 +797,20 @@ def command_output_from_subprocess(self , args, process_wait_time):
         out = str(out)
         return out
 
+    def get_severity_level(self):
+        logging_level = LoggingLevel(LoggingConstants.DefaultEventLogLevel)
```

Comment:
Is the default level set for 2? If not, do we need to create a separate constant for guest agent Kusto Logs?

Reply:
Yes, it is set to 2